docs/api/categories.md

```diff
@@ -0,0 +1,265 @@
+# Cagegories
+
+Categorize Speech Challenges or categories.
+
+## List all top level categories
+
+List all top level categories which do not have a parent category.
+
+### URL
+
+```http
+GET /categories HTTP/1.1
+```
+
+### Request
+
+```http
+GET /categories HTTP/1.1
+Accept: application/json
+```
+
+### Response
+
+The resonse is a JSON list with categories.
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "id": "category_1",
+    "created": "2017-01-12T09:35:20Z",
+    "updated": "2017-01-12T09:35:20Z",
+    "name": "Category 1",
+    "description": "Some awesome description.",
+    "color": "#00f",
+    "imageUrl": "https://api.itslanguage.nl/download/UKbsMpBsXaJUsBbK",
+    "iconUrl": "https://api.itslanguage.nl/download/GdExSbs-ZVNnQUUe",
+    "categories": ["category_1_1", "category_1_2"],
+    "speechChallenges": [],
+    "progress": {
```

Comment:
Now we're mixing personal progress with the generic category data in one API call. Not sure if that's a good idea. It might be, but it's a new idea so I have to think about it. Because it makes caching API calls harder, I lean towards an extra API call to get your personal progress, but I'm not fully leaning one way or the other yet.